core/src/components/modal/modal.tsx

```diff
@@ -756,7 +776,7 @@ export class Modal implements ComponentInterface, OverlayInterface {
           part="content"
           ref={el => this.wrapperEl = el}
         >
-          {showHandle && <div class="modal-handle" part="handle"></div>}
+          {showHandle && <div class="modal-handle" part="handle" onClick={this.onHandleClick}></div>}
```

Comment:
Since this is not a button, does Axe or other assistive tech not like this?

Reply:
Will confirm, I would assume it is an issue. Should be a button based on the revised behavior. Good catch 👍